.github/scripts/ep_review.py

```
@@ -0,0 +1,151 @@
+from pathlib import Path
+
+from ep_hooks import EPHooks
+from ep_skill_config import build_skill_config
+
+
+REPO = "osac-project/enhancement-proposals"
+SKILLS_PATH = "/opt/skills"
+
+
+def gh(args):
+    result = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)